.gitignore

```diff
@@ -25,4 +25,5 @@ nuget.exe
 *.*sdf
 *.ipch
 project.lock.json
-.vs
\ No newline at end of file
+.vs
+*.linemappings.cs
```

Comment:
Nahhhh, undo. It makes it easier to find out which line mappings have changed by having it show up in git. Otherwise you have to search for it.